source/main_module/main.c

```diff
@@ -76,7 +78,7 @@ ADCChannelConfig_t adc_channel_config[] = {
     {.channel=I_SENSE_C1_ADC_CHNL,    .rank=5, .sampling_time=ADC_CHN_SMP_CYCLES_6_5},
     {.channel=LV_I_SENSE_ADC_CHNL,    .rank=6, .sampling_time=ADC_CHN_SMP_CYCLES_6_5},
 };
-dma_init_t adc_dma_config = ADC1_DMA_CONT_CONFIG((uint32_t) &adc_readings, 
+dma_init_t adc_dma_config = ADC1_DMA_CONT_CONFIG((uint32_t) &adc_readings,
```

Comment:
Just a general thing, don't commit lines that didn't necessarily change. When someone goes to view the history of this file, it'll look like these lines were changed for the fault library, but the fault library has nothing to do with the ADC.

Reply:
Will do, to be honest, idk why that was ever recorded as being changed, as don't remember ever touching it. It was probably accidental and got committed when I added the node defs to the various main.cs or something similar.